src/main/java/org/mifos/connector/mockPaymentSchema/api/BatchApi.java

```diff
@@ -0,0 +1,34 @@
+package org.mifos.connector.mockPaymentSchema.api;
+
+import org.mifos.connector.mockPaymentSchema.schema.BatchDTO;
+import org.mifos.connector.mockPaymentSchema.schema.BatchDetailResponse;
+import org.mifos.connector.mockPaymentSchema.schema.Transfer;
+import org.mifos.connector.mockPaymentSchema.service.BatchService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("mockapi/v1")
+public class BatchApi {
+
+    @Autowired
+    private BatchService batchService;
+
+    @GetMapping("/batch/summary")
```

Comment:
I understand that these APis are copied from the Operations app, yet we need to update them based on the REST API standards.